services/statsService.js

```diff
@@ -57,38 +57,44 @@ ORDER BY
   try {
     const customStats = await getCustomStats();
 
-    const overriddenData = data.map(d => {
-      const customCountryStat = customStats.find(c => c.countryCode && d.countryCode && c.countryCode.toLowerCase() === d.countryCode.toLowerCase());
+    const overriddenData = [];
+    if(date) {
+      overriddenData = data;
+    }
+    else {
+      overriddenData = data.map(d => {
```

Comment:
Hi @clearsense, `const` variable declaration cannot be reassigned, use `let` instead.

Reply:
Yeah, sure. Done